www/src/pages/de/usage/typescript.md

```diff
@@ -31,33 +31,33 @@ lang: en
   </cite>
 </blockquote>
 
-Whether you're a new or seasoned developer, we think that TypeScript is a must have. It can look intimidating at first, but much like a lot of tools, is something that many never look back from after starting to use it.
+Egal ob du ein neuer oder erfahrener Entwickler bist, wir denken, dass TypeScript ein Muss ist. Es kann zu Beginn etwas abschreckend wirken, aber wie bei vielen Tools, ist es etwas, das viele nie wieder missen wollen, nachdem sie damit angefangen haben es zu verwenden.
 
-It provides live feedback as you write your code by defining expected data types, and either provides helpful autocomplete in your code editor, or yells at you with red squiggly lines if you're trying to access a property that doesn't exist or trying to pass a value of the wrong type, which you would otherwise have to debug further down the line.
+Es bietet Live-Feedback während du deinen Code schreibst. Die erwarteten Datentypen werden definiert und entweder erhalten wir hilfreiche Autovervollständigung in unserem Code-Editor oder es wird uns mit roten Unterstrichen auf ein Problem aufmerksam gemacht z.B. wenn wir versuchen auf eine Eigenschaft zuzugreifen, die nicht existiert oder wenn wir versuchen einen Wert eines falschen Typs zu übergeben. Dadurch können wir Fehler frühzeitig erkennen und beheben ohne erst im Nachhinein debuggen zu müssen.
```

Comment:
not so happy with that block, ideas?

Reply:
what do you think about this?

Es gibt dir Live-Feedback während du deinen Code schreibst, indem erwartete Datentypen definiert werden. 
Dieses zeigt sich durch die autocomplete-Funktion oder durch rote wellige Linien in deinem Editor. Beispielsweise wenn du versuchst auf einen Wert zuzugreifen, der nicht existiert, oder wenn du versuchst einen Wert zuzuweisen, der den falschen Typ hat. Beide Fälle müssten sonst gegebenenfalls später debuggt werden.

Reply:
good suggestion, I guess we can combine both of our suggestions. 
curious what other think tho

Reply:
I generally enjoy a more "personal" approach when writing docs. So I generally try to speak to the reader with things like "dir" etc. So I like the suggestion. anOatFlake made.